src/modules/translation/locales/es.json

```diff
@@ -1 +1,10 @@
-{}
+{
```

Comment:
The avatar edit translation is missing in the Spanish and the Chinese versions, would you mind adding them?